enhancements/installer/ibm-cloud-for-power-vs-ipi.md

```diff
@@ -256,9 +256,27 @@ as because it is managing DNS & the Load Balancer (which is already set up with
 the specifics of these interactions is available in [JIRA](<https://issues.redhat.com/browse/MULTIARCH-1103>).
 
 ### Persistent Storage
+The support for PowerVS storage is configured as day 1 operation. OCP ships a new cluster operator called ibm-powervs-block-csi-driver-operator.
+1. [ibm-powervs-block-csi-driver](https://github.com/kubernetes-sigs/ibm-powervs-block-csi-driver) is downstreamed to [openshift](https://github.com/openshift/ibm-powervs-block-csi-driver)
+2. The ibm-powervs-block-csi-driver-operator installs the IBM PowerVS Block CSI driver following the  [recommended guidelines](https://github.com/openshift/enhancements/blob/master/enhancements/storage/csi-driver-install.md)
+3. The ibm-powervs-block-csi-driver-operator deploys all the objects required by the  IBM PowerVS Block CSI driver:
+   1. A namespace called `openshift-cluster-csi-drivers`
+   2. Two ServiceAccounts: one for the Controller Service and other for the Node Service of the CSI driver
+   3. The RBAC rules to be used by the sidecar containers
+   4. The CSIDriver object representing the CSI driver
+   5. A Deployment that runs the driver's Controller Service
+   6. A DaemonSet that runs the driver's Node Service
+   7. A non-default StorageClass that uses the CSI driver as a provisioner.
```

Comment:
Why is the storage class non-default? Will there be other CSI driver that will provide the default one?

Reply:
May it is typo.. this will be a default storage class for the IPI deployment..

Reply:
It was a typo. Fixed it now.